libxrdp/libxrdp.c

```diff
@@ -135,18 +131,18 @@ libxrdp_force_read(struct trans* trans)
 
     if (trans_force_read(trans, 4) != 0)
     {
-        g_writeln("libxrdp_force_read: header read error");
+        LOG(LOG_LEVEL_WARNING, "libxrdp_force_read: header read error");
```

Comment:
Thank you very much for your careful work! You did this not just by replacing them with a single log level but considered the log context.